File: benchmarks/suite/list_ops.c
#include <stdio.h>
int main(void) {
    int nums[20];
    for (int k = 0; k < 20; k++) nums[k] = k;
    long long sum = 0;
    for (int i = 0; i < 2000000; i++)
        for (int k = 0; k < 20; k++)
            sum += nums[k];
    printf("%lld\n", sum);
    return 0;
}

File: benchmarks/suite/list_ops.go
package main

import "fmt"

func main() {
	nums := make([]int, 20)
	for k := range nums {
		nums[k] = k
	}
	var sum int64
	for i := 0; i < 2_000_000; i++ {
		for _, v := range nums {
			sum += int64(v)
		}
	}
	fmt.Println(sum)
}
